src/dotnet/updater.py

```diff
@@ -432,7 +422,7 @@ def _is_latest_dotnet(version: _DOTNET_VERSION_T, os_version: OsVersion) -> bool
                 os_version=os_version,
                 name="dotnet-aspnet",
                 is_sdk=False,
-                pretty_name=f"ASP.NET {ver} runtime",
+                pretty_name=f"ASP.NET Core Runtime {ver}",
```

Comment:
we use sentence style capitalization here

Reply:
these are the official names and case styling used by Microsoft.

Reply:
See this for example https://dotnet.microsoft.com/en-us/download/dotnet/8.0

I don't see a reason to use lowercase on the package name, we don't call Apache tomcat, ASP.NET Core Runtime is the entire name, so the image would still be "ASP.NET Core Runtime container image".

Reply:
@dmpop or @lvicoun can you advise what is best here?

Reply:
Easy: follow the capitalization used in the official product name. :smile: